doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-GIVEN-proc.adoc

```diff
@@ -4,10 +4,10 @@
 The *GIVEN* section defines input facts for the test. For example, if an `Underage` rule in the project declines loan applications for applicants under the age of 21, then the *GIVEN* facts in the test scenario could be `Applicant` with `age` set to some integer less than 21.
 
 .Prerequisite
-All data objects required for your test scenario have been created or imported and are listed in the *Data Objects* tab of the test scenarios designer.
+All data objects required for your test scenario have been created or imported and are listed in the *Data Objects* tab of the legacy test scenarios designer.
 
 .Procedure
-. In the test scenarios designer, click *GIVEN* to open the *New input* window with the available facts.
+. In the legacy test scenarios designer, click *GIVEN* to open the *New input* window with the available facts.
```

Comment:
Minor suggestion - to make the reference consistent. Maybe check if there is a label in the UI?

- legacy test scenarios designer
- *Test Scenario (Legacy)* designer

Reply:
Changed to UI label as suggested.